fontforgeexe/openfontdlg.c

```diff
@@ -135,11 +146,7 @@ struct openfilefilters def_font_filters[] = {
     { NU_("ΤεΧ Bitmap Fonts"), "*{pk,gf}" },
     { N_("PostScript"), "*.{pfa,pfb,t42,otf,cef,cff,gai,pf3,pt3,gsf,cid}{.gz,.Z,.bz,.bz2,.lzma,}" },
     { N_("TrueType"), "*.{ttf,t42,ttc}{.gz,.Z,.bz,.bz2,.lzma,}" },
-#ifdef _NO_LIBPNG
```

Comment:
What obsoleted the NO_LIB_PNG logic? And how did it relate to woff?

Reply:
I'm guessing that the `_NO_LIBPNG` check is because the woff (not woff2) code depends on (maybe?) zlib compression that would have been present with libpng. Or something like that.

Reply:
Is that obsolete now? Does plain WOFF fail safely elsewhere without libpng?

Reply:
No, it's still needed for the WOFF code. https://github.com/fontforge/fontforge/blob/master/fontforge/woff.c#L67

Reply:
Okay. I guess moving it is the best approach then.

Reply:
Some time ago, someone indicated zlib is pretty well a requirement, so fontforge requires zlib. libpng depends on zlib, but doesn't necessarily need to be included.
WOFF requires zlib, but appears  has nothing to do with png, so it would appear ok to leave-out the lib_png dependency when speaking of WOFF. I just glanced quickly through this page but didn't dig in... https://www.w3.org/blog/2018/03/woff-2-0-the-inside-scoop/

Reply:
Good point, I see it's a hard dependency [here](https://github.com/fontforge/fontforge/blob/master/configure.ac#L371), I'll remove conditionals on LIBPNG where it only requires zlib.